docs/specifications/1-introduction.md

```diff
@@ -0,0 +1,172 @@
+# Introduction
+
+This document provides a high level overview of *jsii*, starting with it's
+design tenets. It introduces the concepts and components that compose *jsii*.
+
+
+## Updating the Specification
+
+### Introduction
+
+The *jsii* specification follows the guiding principles of an RFC. It is a
+living document that describes the current understanding of how the various
+[components](#components) of *jsii* are operating, as well as the approaches
+used to ensure consistent behavior across the various supported languages.
+
+The document is mastered with the *jsii* codebase, making it easy to determine
+what specification was in place at the time of a give *jsii* release (by ways of
+referring to the `vX.Y.Z` git tag). A single version of the specification is
+considered **active** at any given time: the version of the specification that
+is represented on the `HEAD` commit of the `master` branch of the [`aws/jsii`]
+repository. The **active** specification must be the base version for any update
+proposal.
```

Comment:
This implies (and I like it) that the spec describes the *current* state of the project.

(we can define some documentation convention to annotate a paragraph/section and indicate what would be the desired behavior/spec that will be implemented in the future, maybe "not implemented yet").

Reply:
Yeah I should add a note about the block quotes I have been using to denotes that, and enshrine it as "the way to go" (I find it works nicely in the document flow)